data/search.go

```diff
@@ -69,12 +69,12 @@ func (r *SeriesRepository) GetSearchSummary(searchText string) (searchSummary mo
 
 	rows, err := r.DB.Query(`SELECT geographies.fips, geographies.display_name_short, geofreq.geo, geofreq.freq
 FROM (SELECT MAX(SUBSTRING_INDEX(SUBSTR(s.name, LOCATE('@', s.name) + 1), '.', 1)) as geo, MAX(RIGHT(s.name, 1)) as freq
-      FROM (SELECT name FROM series
+      FROM (SELECT series.name FROM series
```

Comment:
Gah! Somehow when I was reviewing the code this possibility didn't occur to me :=( Is this really the only place it happens?